src/styles/_button.scss

```diff
@@ -0,0 +1,24 @@
+.button {
+  @include center;
+  @include roboto700;
+
+  height: 40px;
+  width: 100%;
+  margin-top: 16px;
+  border-radius: 5px;
+  border-color: 1px solid #00acdc;
```

Comment:
The `border-color` property is incorrectly used here. It should be `border: 1px solid #00acdc;` to define the border width, style, and color together. This will ensure the border is displayed correctly.